schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL,
+    species VARCHAR(100),
     PRIMARY KEY (id)
 );
```

Comment:
Please add a new column called `species` with a string type in the schema.sql file as per project requirements.
![image](https://user-images.githubusercontent.com/38088818/171428208-aeb11dd4-bf48-461e-b03e-4edba2234c9a.png)

```
ALTER TABLE animals ADD species varchar(255);
```